database/other/communities.json

```
@@ -54,5 +54,12 @@
     "url": "https://vexpert.vmware.com/",
     "category": "other",
     "subcategory": "communities"
+  },
+  {
+    "name": "Women Who Code",
+    "description": "Women Who Code is a global non-profit organization dedicated to change the industry experience for women engineers. It is an active community that aims to provide an avenue into the technology world by evaluating and assisting women in developing technical skills.",
```

[suggestion] I'm a member of one of their communities and they always strive to create inclusive environments for women and non-binary individuals from different backgrounds, so I recommend mentioning this point as shown in the example below

```suggestion
    "description": "Women Who Code is a global non-profit organization dedicated to providing women and non-binary individuals from diverse backgrounds with the necessary tools and skills to thrive in the tech industry.",
```

---

@CBID2 thanks for mentioning it , just made the changes  in the description. Please have a look .